content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +93,186 @@ there are any errors on the work entry, such as required fields being left blank
 required to be resolved before payslips can be generated.
 
 Any work entry that has a conflict to be resolved is indicated on the main :guilabel:`Work Entry`
-overview dashboard, located by going to :menuselection:`Payroll --> Work Entries --> Conflicts`,
-where only conflicts needing resolution are shown.
+dashboard. Navigate to :menuselection:`Payroll --> Work Entries --> Work Entries`, and  only
+conflicts needing resolution are shown by default.
 
-.. image:: work_entries/conflicts.png
-   :align: center
-   :alt: Conflicts dashboard view showing all employee's conflicts in work entries.
-
-Conflicts are indicated with an orange triangle in the top left corner of each individual entry.
-Click on an individual work entry to see the conflict details in a pop-up.
+Conflicts are indicated with an orange triangle in the top left corner of each individual work
+entry. Click on an individual work entry to see the date and time for the specific work entry, then
+click :guilabel:`Edit` to view the conflict details in a pop-up.
 
 The conflict is briefly explained in an orange text box.
 
-.. image:: work_entries/conflict-detail.png
-   :align: center
-   :alt: Details for a conflict appear in the pop-up.
+The :guilabel:`Description`, :guilabel:`Employee`, and :guilabel:`Work Entry Type` are listed on
+the left side of the pop-up. The :guilabel:`From` and :guilabel:`To` date and time range, as well as
+the total time (in hours), appears in the :guilabel:`Duration` field on the right side.
+
+If the conflict is due to a time-off request that has not been approved yet, a :guilabel:`Time Off`
+field appears on the left side, with the type of time off requested in the description.
+
+Time off conflicts
+------------------
+
+The most common work entry conflicts are for time off requests that have been submitted but not yet
+approved, which result in duplicate work entries for that employee (one for time-off and another for
+regular work). If there is a conflict because a time-off request is in the system for the same time
+that a regular work entry already exists, the time-off request is entered in the :guilabel:`Time
```

Comment:
```suggestion
regular work). If there is a conflict because a time off request is in the system for the same time
that a regular work entry already exists, the time off request is entered in the :guilabel:`Time
```